build.gradle

```diff
@@ -70,12 +72,6 @@ dependencyRecommendations {
     propertiesFile file: project.rootProject.file('versions.props')
```

Comment:
you'll want to remove this too, and add `com.palantir.baseline-versions.disable = true` to gradle properties

Reply:
Ah, thanks!